examples/hosts/app-integration/external-data/src/utilities/webhook.ts

```diff
@@ -70,8 +70,20 @@ export class MockWebhook<TData = unknown> {
 			`EXTERNAL DATA SERVICE WEBHOOK: External data has been updated. Notifying ${this._subscribers.size} subscribers...`,
 		);
 
-		const messageBody = JSON.stringify({ data });
 		for (const subscriberUrl of this._subscribers) {
+			console.log(`EXTERNAL DATA SERVICE WEBHOOK: Notifying ${subscriberUrl}`);
+			// TODO: use a query string parser here instead of this hacky lookup of '=externalTaskListId'
+			// Tried using node:querystring and node:url but that is not allowed by
+			// the rules and haven't found another one that works in a simple search so far.
+			const externalTaskListId = subscriberUrl.slice(
```

Comment:
Might also be good to say that this could break if any additional querystring parameters are added to that URL.